Fixed broken links

ryanshripat opened the pull request:
Most of the links were pointing to articles inside of https://github.com/MicrosoftDocs/azure-docs/blob/master/azure/cognitive-services/Face/Tutorials/, but these were resulting in 404s. All links using /azure/cognitive-services have now been updated to use /articles/cognitive-services instead. Additionally, the .md extension missing from these links were added.

azurecla commented:
Hi __@yamatoya__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>You've already signed the contribution license agreement. Thanks!</span>
        <p>The agreement was validated by Microsoft and real humans are currently evaluating your PR.</p>

TTYL, AZPRBOT;

ryanshripat committed:
Fixed broken links

Most of the links were pointing to articles inside of https://github.com/MicrosoftDocs/azure-docs/blob/master/azure/cognitive-services/Face/Tutorials/, but these were resulting in 404s. All links using /azure/cognitive-services have now been updated to use /articles/cognitive-services instead. Additionally, the .md extension missing from these links were added.
